### PR TITLE
Changes for Payment Views for PAD settlement

### PR DESCRIPTION
**Pull request by saravanpa-aot**
* Added the new routes for pad settling payment by cc

*Issue #:*
https://github.com/bcgov/entity/issues/4920

*Description of changes:*

Just added the necessary routes and api calls to do the CC settlement

Pending:

Work on the UI changes for refreshing the org status till it gets the correct status from backend.

*Checklist:*
I confirm that below checklist items are addressed in this pull request; (check the boxes applicable)
- [ ] No lint errors
- [ ] No test case failures and proper coverage for all modules/classes
- [ ] Updated the deployment configs for new environment variable(s)
- [ ] Updtaed the postman collection in entity repository (https://github.com/bcgov/entity/tree/master/api-e2e/postman) for e2e tests

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-auth license (Apache 2.0).

### CODE REVIEW

File: auth-web/src/routes/router.ts [hunk 1]
```diff
@@ -372,6 +374,13 @@ export function getRoutes (): RouteConfig[] {
       props: true,
       meta: { requiresAuth: false }
     },
+    {
+      path: '/make-pad-payment/:paymentId/transactions/:redirectUrl',
```
Comment (sumesh-aot) [inline auth-web/src/routes/router.ts:378]: can we have a better route? Technically, this is not a PAD payment.. may be make-cc-payment ?

**Review by sumesh-aot — APPROVED** [review]
Please check comment about the route name

File: auth-web/src/routes/router.ts (hunk 1 above)
Comment (saravanpa-aot) [reply auth-web/src/routes/router.ts:378]: yup..wil change it

File: auth-web/src/components/auth/account-freeze/PaymentReview.vue [hunk 2]
```diff
@@ -78,10 +86,14 @@ import Steppable from '@/components/auth/common/stepper/Steppable.vue'
 })
 export default class PaymentReview extends Mixins(Steppable) {
   private readonly currentOrganization!: Organization
+  private readonly createAccountPayment!: () => Payment
   private isAcknowledged: boolean = false
 
-  private proceed () {
-    // proceed final step
+  private async proceedToPayment () {
+    const payment:Payment = await this.createAccountPayment()
```
Comment (nitheesh-aot) [inline auth-web/src/components/auth/account-freeze/PaymentReview.vue:93]: can we move this to the view file of this stepper?

do emit('final-step-action') in this function.
and handle that in the Stepper view.(parent)

similar to other steppers.

**Review by nitheesh-aot — CHANGES_REQUESTED** [review]

File: auth-web/src/components/auth/account-freeze/PaymentReview.vue (hunk 2 above)
Comment (saravanpa-aot) [reply auth-web/src/components/auth/account-freeze/PaymentReview.vue:93]: okay..didnt know we have to do..

File: auth-web/src/routes/router.ts (hunk 1 above)
Comment (saravanpa-aot) [reply auth-web/src/routes/router.ts]: done

File: auth-web/src/components/auth/account-freeze/PaymentReview.vue (hunk 2 above)
Comment (saravanpa-aot) [reply auth-web/src/components/auth/account-freeze/PaymentReview.vue]: Done.Do you mind having a look?

**Review by nitheesh-aot — APPROVED** [review]

**Pull request merged**